dependencies.tsv

```diff
@@ -4,12 +4,12 @@ github.com/juju/errors	git	7a748941731116a391d328fd2ba9744f57efb51b	2014-12-11T1
 github.com/juju/gojsonpointer	git	0154bf5a168b672d8c97d8dd83a54cb60cd088e8	2014-07-18T03:59:30Z
 github.com/juju/gojsonreference	git	0673d58f64bacac2db34c7d1e87a599d58923981	2014-07-18T03:57:39Z
 github.com/juju/gojsonschema	git	33fa79718fa9b24e2a04122f91a75496c0f77098	2014-07-17T16:12:25Z
-github.com/juju/juju	git	a1a44269e53e2eb99abb97e17adfeff4f195b41f	2014-12-15T05:01:35Z
+github.com/juju/juju	git	a646f6164498c5b5deca7f02223f62b93f9eb5ea	2015-03-24T21:57:38Z
```

Comment:
I don't want to update this dependency until pulling in the service package doesn't drag in testing deps too.